README.md

```diff
@@ -27,7 +27,7 @@ spec:
 
 ![a relative link](docs/assets/operatorhub-listing.png)
 
-3. Install the operator using the defaults in the wizard, and wait for it to show up in the list of "Installed Operators". If it doesn't install properly, you can check on its status in the "Installed Operators" tab in the `openshift-operators` namespace.
+3. Install the operator in the `openshift-gitops-operator` namesapce using the defaults in the wizard, and optionally, select the checkbox to enable cluster monitoring on the namesapce. Wait for it to show up in the list of "Installed Operators". If it doesn't install properly, you can check on its status in the "Installed Operators" tab in the `openshift-gitops-operator` namespace.
```

Comment:
@jaideepr97 Do we have any possible installation failures and underlying causes or symptoms for the same? If yes, can we have troubleshooting tips for them? Thanks!

Reply:
@jaideepr97 I see that the user has to just select/enable the OpenShift monitoring option while installing the GitOps Operator in the `openshift-gitops-operator` namespace.

- Do we need any documentation or information like a concept/overview, or anything that we need in the Downstream documentation from the user's perspective, about monitoring metrics?

Reply:
@jaideepr97 Can you please respond to my queries here? Thanks!

Reply:
@Srivaralakshmi  I don't know what the best place to put this info is so I'm just putting it here :


Installation instructions to be able to use this feature:
- fresh installations of the operator will see the option to choose the namespace to install the operator in, with the recommended option already selected as `openshift-gitops-operator` namespace 
- users must select the check box saying `Enable Operator recommended cluster monitoring on this Namespace` to enable cluster monitoring 

![ns-installation](https://github.com/redhat-developer/gitops-operator/assets/21956986/bff62209-2fde-456b-9598-6336e007f076)




Usage instructions:
- Once operator is installed, users can leverage the  `Observe > metrics` page in the admin view to query the metrics exposed by the operator 
- At present the new operator performance metrics exposed are:
 - `active_argocd_instances_total` [_Guage_] - This metric produces the graph that tracks the total number of active argo-cd instances being managed by the operator at a given time
 - `active_argocd_instances_by_phase{phase=<phase>}` [_Guage_] - This metric produces the graph that tracks the count of active Argo CD instances by their phase [Available/Pending/Failed/unknown]
 - `active_argocd_instance_reconciliation_count{namespace="<argocd-instance-ns>"}` [_Counter_] - This metric produces the graph that tracks total number of reconciliations that have occurred for the instance in the given namespace at any given point in time
 - `controller_runtime_reconcile_time_seconds_per_instance_bucket{namespace="<argocd-instance-ns>",le="0.5"}` [_Histogram_]- This metric tracks the number of reconciliations that took under 0.5s to complete for a given instance
 
 Any of these metrics can be queried against prometheus as described above, or can be used for more complex queries as is true with any other promql queries 

![reconciliations_per_instance](https://github.com/redhat-developer/gitops-operator/assets/21956986/3a81948b-bef9-4a6b-aa31-689d53032827)
![argocd_instances_by_phase](https://github.com/redhat-developer/gitops-operator/assets/21956986/26e30f61-1204-4d08-998e-7cf57bcfd803)
![active_argocd_instances_total](https://github.com/redhat-developer/gitops-operator/assets/21956986/fc4d2b63-e549-41dc-994b-9bb9968d731c)



>